sympy/solvers/ode/tests/test_ode.py

```diff
@@ -644,22 +646,14 @@ def test_dsolve_linsystem_symbol_piecewise():
     u = Symbol('u')  # XXX it's more complicated with real u
     eq = (Eq(diff(f(x), x), 2*f(x) + g(x)),
            Eq(diff(g(x), x), u*f(x)))
-    s1 = [Eq(f(x), Piecewise((C1*exp(x*(sqrt(4*u + 4)/2 + 1)) +
```

Comment:
I guess that the XXX comment no longer applies

Reply:
Examples like this that are solved by the neq solver should probably be moved to `test_system`. It would be good to add a comment noting that this used to give a Piecewise but now does not. Perhaps this example can go with the regression test for the issue about Piecewise.